src/2982R0_std_quantity_as_a_numeric_type.md

```diff
@@ -991,18 +993,136 @@ The above program will produce the following types for acceleration quantities
 
 ## Scaled units
 
+Another very common operation is to multiply an existing unit by a factor, creating a new, scaled unit.  For
+example, the unit _foot_ can be multiplied by 3, producing the unit _yard_.
+
+The process also works in reverse: the ratio between any two units of the same dimension is a well-defined
+number.  For example, the ratio between one foot and one inch is 12.
+
 ### Unit magnitudes
 
-TBD
+In principle, this scaling factor can be any positive real number.  In mp-units and Au, we have used the term
+"magnitude" to refer to this scaling factor.  (This should not be confused with the logarithmic "magnitude"
+unit commonly used in astronomy.)
 
-### Construction helpers
+In the library implementation, each unit is associated with a magnitude.  However, for most units the
+magnitude is a fully encapsulated implementation detail, not a user-facing value.
 
+This is because the notion of "the" magnitude of a unit is not generally meaningful: it has no physically
+observable consequence.  What _is_ meaningful is the _ratio_ of magnitudes between two units of the same
+dimension.  We could associate the _foot_, say, with any magnitude $m_f$ that we like --- but once we make that
+choice, we must assign $3m_f$ to the _yard_, and $m_f/12$ to the _inch_.  Separately and independently, we can
+assign any magnitude $m_s$ to the second, because it's an independent dimension --- but once we make that
+choice, it fixes the magnitude for derived units, and we must assign, say, $(5280 m_f) / (3600 m_s)$ to the
+_mile per hour_.
 
+The one exception to the arbitrariness of magnitudes is _dimensionless_ units.  Because their dimension is
+null, quantities of these units can be meaningfully compared to their squares and other powers.  For example,
+the magnitude of _percent_ is $1 / 100$, and the magnitude of _squared percent_ (or _pertenk_, "per-10-k") is
+$1 / 10000$.  We cannot choose another value for the magnitude without producing observably incorrect results.
 
-### Common units
+### Requirements and Representation
+
+A magnitude is a positive real number.  The best way to _represent_ it depends on how we will _use_ it.  To
+derive our requirements, note that magnitudes must support every operation which units do.  This is because
+units have two components that participate in operations: _dimension_, and _magnitude_.  When we apply an
+operation to units, we simply apply it separately to the dimension and magnitude parts.
+
+Units are closed under _products_ and _rational powers_.  Therefore, our magnitude representation must support
+these operations natively and robustly; this is the most basic requirement.  We must also support certain
+_irrational_ "ratios", such as the factor of $\frac{\pi}{180}$ between _degrees_ and _radians_.
+
+The usual approach, `std::ratio`, fails to satisfy these requirements in multiple ways.
+
+- It is not closed under rational powers (rather infamously, in the case of $2^\frac{1}{2}$).
```

Comment:
I'm changing this to `2<sup>1/2</sup>` (2<sup>1/2</sup>).
This is how it currently looks on my system:
![1697985431](https://github.com/mpusz/wg21-papers/assets/21071787/71f6d23f-1e9c-473f-b5f2-b00ca7930ad2)